config/generate_repository_config.yaml

```diff
@@ -68,3 +68,8 @@ mysql:
     deb:
       path: http://repo.mysql.com/apt/
       key: "5072E1F5"
+clustrix:
+  releases:
+    9.1.4:
+      path: http://files.clustrix.com/releases/software/clustrix-9.1.4.el7.tar.bz2
```

Comment:
I was able to download previous releases like 9.1.3 and 9.1.2 just by changing the corresponding number. I think that direct pasting of the path breaks the whole idea of the repository generator.